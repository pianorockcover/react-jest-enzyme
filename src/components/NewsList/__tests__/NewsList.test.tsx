import { shallow } from "enzyme"
import React from "react";
import { NewsList, NewsListProps } from "..";

describe("Test components", () => {
    it("Test NewsList component render", () => {
        const props: NewsListProps = {
            news: [
                {
                    id: 1,
                    title: "title",
                    body: "body"
                }
            ]
        }

        const copomnent = shallow(<NewsList { ...props } />);
        expect(copomnent).toMatchSnapshot();
    })
})