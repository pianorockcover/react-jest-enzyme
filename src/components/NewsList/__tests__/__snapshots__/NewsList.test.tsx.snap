// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test components Test NewsList component render 1`] = `
<div
  data-news-list={true}
>
  <div
    className="card"
    data-news-card={true}
    key="0"
  >
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        title
      </h5>
      <div>
        body
      </div>
    </div>
  </div>
</div>
`;
